Sources/Overload/OvEditor/src/OvEditor/Panels/Console.cpp

```diff
@@ -100,6 +101,11 @@ void OvEditor::Panels::Console::OnLogIntercepted(const OvDebug::LogData & p_logD
 	consoleItem1.enabled = IsAllowedByFilter(p_logData.logLevel);
 
 	m_logTextWidgets[&consoleItem1] = p_logData.logLevel;
+
+	if (m_logGroup->GetWidgets().size() > Settings::EditorSettings::ConsoleMaxLogs)
+	{
+		m_logGroup->RemoveWidget(*m_logGroup->GetWidgets().front().first);
```

Comment:
What happens if there are already 100 logs in the console, and the user set the max to 10? I feel like the user will always have at least as many logs as he already had before he set the limit